tests/tests/User/Notification/UserNotificationEventHandlerTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Concrete\Tests\User\Notification;
+
+use Carbon\Carbon;
+use Concrete\Core\Entity\Notification\UserDeactivatedNotification;
+use Concrete\Core\Entity\User\User;
+use Concrete\Core\Notification\Notifier\NotifierInterface;
+use Concrete\Core\Notification\Notifier\StandardNotifier;
+use Concrete\Core\Notification\Subscription\SubscriptionInterface;
+use Concrete\Core\Notification\Type\Manager;
+use Concrete\Core\Notification\Type\UserDeactivatedType;
+use Concrete\Core\User\Event\DeactivateUser;
+use Concrete\Core\User\Notification\UserNotificationEventHandler;
+use Hamcrest\Core\IsEqual;
+use Hamcrest_Core_IsEqual;
+use Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;
+use PHPUnit_Framework_TestCase;
+use Mockery as M;
+
+class UserNotificationEventHandlerTest extends PHPUnit_Framework_TestCase
+{
+
+    use MockeryPHPUnitIntegration;
+
+    public function testHandlingEvent()
+    {
+        if (!class_exists(IsEqual::class)) {
```

Comment:
I'd add this code to the tests bootstrap.php file, so that we don't have to implement this code in other classes too.

Reply:
I will if I need to use this again. v9 won't have this issue because it will have newer dependencies.